tests/admin/test-class-boldgrid-backup-admin-premium-features-page.php

```diff
@@ -76,14 +77,14 @@ public function test_admin_enqueue_scripts() {
 
 	}
 
-	public function test_page() {
-		$this->premium_page->core->plugin->pluginData = $this->get_plugin_data( '2.1.1' );
+	// public function test_page() {
```

Comment:
Should this be commented out?

Reply:
Same as above. Should have been removed. now it has been